app/View/Pages/home.ctp

```diff
@@ -79,7 +79,7 @@ endif;
 		echo '<span class="notice">';
 			echo __d('cake_dev', 'Your database configuration file is NOT present.');
 			echo '<br/>';
-			echo __d('cake_dev', 'Rename %s to %s', 'APP/Config/database.php.default', 'APP/Config/database.php');
+			echo __d('cake_dev', 'Rename %s to %s', 'CONFIG/database.php.default', 'CONFIG/database.php');
```

Comment:
Same as in https://github.com/cakephp/cakephp/pull/10579/files#diff-17d0cb9c5cd4a62e574aa3dc8083249eR61